src/components/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import styles from './Header.module.scss'
+import logo from '../../img/logo.png'
+
+import NavBar from '../Navbar/Navbar'
+
+class Header extends React.Component {
```

Comment:
Rename component name to 'PageHeader', cause we can have different headers in our app.